openapi.json

```diff
@@ -0,0 +1,9995 @@
+            "nullable": true,
+            "title": "dd_enabled",
+            "type": "boolean",
+            "x-nullable": true
+          },
+          "foundation_date": {
+            "description": "Company Foundation Date",
+            "format": "date",
+            "title": "foundation_date",
+            "type": "string"
+          },
+          "hr_registration_date": {
+            "description": "to be documented",
+            "format": "date",
+            "nullable": true,
+            "title": "hr_registration_date",
+            "type": "string",
+            "x-nullable": true
+          },
+          "hr_registration_number": {
+            "description": "to be documented",
+            "nullable": true,
+            "title": "hr_registration_number",
+            "type": "string",
+            "x-nullable": true
+          },
+          "id": {
+            "description": "to be documented",
+            "readOnly": true,
+            "title": "id",
+            "type": "integer"
+          },
+          "last_payment_amount": {
+            "description": "Ammount of last known payment (on account level)",
+            "format": "float",
+            "nullable": true,
+            "title": "last_payment_amount",
+            "type": "number",
+            "x-nullable": true
+          },
+          "legal_form": {
+            "description": "Legal form of Company",
+            "enum": ["Limited", "Listed", "Single Owner"],
+            "title": "legal_form",
+            "type": "string"
+          },
+          "main_stock_exchange": {
+            "description": "Stock Exchange where Company's Ultimate Parent is listed",
+            "title": "main_stock_exchange",
+            "type": "string"
+          },
+          "master_account_id": {
+            "description": "Account FK to Company / Master Account",
+            "title": "master_account_id",
+            "type": "integer"
+          },
+          "max_bca_limit": {
+            "description": "to be documented",
+            "format": "float",
+            "nullable": true,
+            "title": "max_bca_limit",
+            "type": "number",
+            "x-nullable": true
+          },
+          "noga_code": {
+            "description": "Company Industry Classification along NOGA Code standard",
+            "title": "noga_code",
+            "type": "string"
+          },
+          "noga_description": {
+            "description": "Company Industry Classification along NOGA description",
+            "title": "noga_description",
+            "type": "string"
+          },
+          "operational_activity": {
+            "description": "to be documented",
+            "nullable": true,
+            "title": "operational_activity",
+            "type": "string",
+            "x-nullable": true
+          },
+          "parent": {
+            "description": "ID of direct Parent Company / Parent Master",
+            "title": "parent",
```

Comment:
Please clarify: Is `parent` different from `master_account_id` ?
If different, can you describe how?

Reply:
Yes, is different:

master_account_id is the actual primary key of master account and foreign key of basic account and card. Parent on the other hand is a string (and best case in the future an actual ID) of the legal or ultimate parent of the company representing the basic or master account.

Example:
Master Account: Nestle Schweiz AG
Basic Account: Nestle Schweiz Business Travel
Parent Attribute in Basic Account: Nestle Schweiz (if legal (direct) parent) or Nestle Group (if ultimate parent). 

As we know that the Data Quality is miserable, the business people would like to use open string only. However the IT folk rather opt for a real cleanup and real FK ID links. We only will learn in January which way to actually go for. 

If we go for real FK ID links then we will revert to a more sophisticated model, where we distinguish between legal persons and natural persons using a person_type. And then that Parent attribute will always link to the ID of a Legal Person.

All other entities will then be without any company attributes and only the Legal Person records will carry these company specific records and will be linked to Master, Basic and Card.

Reply:
Thanks! We'll treat this as free text until this changes to a FK.